app/experimenter/experiments/tests/test_models.py

```diff
@@ -1064,16 +1065,25 @@ def test_monitoring_dashboard_url_is_valid_when_experiment_not_begun(self):
             ),
         )
 
-    def test_monitoring_dashboard_returns_url_when_rollout(self):
+    @parameterized.expand(
+        [
+            NimbusExperiment.Status.DRAFT,
+            NimbusExperiment.Status.COMPLETE,
+            NimbusExperiment.Status.LIVE,
+            NimbusExperiment.Status.PREVIEW,
+        ]
+    )
+    def test_monitoring_dashboard_returns_url_when_rollout(self, status):
         experiment = NimbusExperimentFactory.create(
             slug="experiment",
-            status=NimbusExperiment.Status.DRAFT,
+            status=status,
             is_rollout=True,
         )
 
         from_date = datetime.date.today() - datetime.timedelta(days=1)
         to_date = datetime.date.today() + datetime.timedelta(days=1)
 
+        self.assertIsNotNone(experiment.monitoring_dashboard_url)
```

Comment:
You don't really need this if you're checking it again on the next line?

Reply:
Oh fair